### ISSUE TITLE
[semantic-issue] Add semantics for sklearn.preprocessing label_binarize

### ISSUE THREAD

**Issue by FrancescoBernini** [spangea/Pyra#150]
**Description**
Notebook: ztgg-project-1.py

**Minimal running example**
```python
from sklearn.preprocessing import label_binarize
y_valid = label_binarize(1, classes=[1, 2, 3, 4, 5])
```

**Links to documentation**
https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.label_binarize.html#sklearn-preprocessing-label-binarize

**Note (optional)**
Binarize labels in a one-vs-all fashion.

Returns:
Y{ndarray, sparse matrix} of shape (n_samples, n_classes)
Shape will be (n_samples, 1) for binary problems. Sparse matrix will be of CSR format.

File ".../lyran/src/lyra/semantics/statistical_type_semantics.py", line 66, in relaxed_open_call_policy
    raise Exception(f"Semantics for {stmt} at line  {stmt.pp.line} not yes implemented")
Exception: Semantics for label_binarize(y_valid) at line  216 not yes implemented
